src/main/java/use_cases/SavePlayer.java

```diff
@@ -0,0 +1,35 @@
+package use_cases;
+
+import entities.HumanPlayer;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class SavePlayer {
+    /**
+     * If the HumanPlayer does not exist in "src/main/java/use_cases/players.csv", saves its name, wins, and losses.
+     * If the HumanPlayer exists in "src/main/java/use_cases/players.csv", do not save anything.
+     * @param player The HumanPlayer object.
+     * @return true if information was saved, false if nothing was saved.
+     * @throws IOException If an input/output error occurs when writing to the file.
+     */
+    public static boolean savePlayerToFile(HumanPlayer player) throws IOException {
```

Comment:
How/where would we handle this exception? That wouldn't be included here but just a thought.

Reply:
The method will throw an IOException if it has an error reading or accessing the file. I don't think this is likely to happen given our relatively simple use of saving/loading data, but I could change this to a catch statement and explicitly handle it although I'm not sure what we would do.